kolibri/auth/api.py

```diff
@@ -184,6 +193,8 @@ def create(self, request):
         serialized_user = self.serializer_class(data=data)
         if serialized_user.is_valid():
             serialized_user.save()
+            serialized_user.instance.set_password(data['password'])
+            serialized_user.instance.save()
```

Comment:
@jamalex Does doing this still log the user out? In fle-internal/kolibri-instant-schools-plugin#62, you did this extra step `update_session_auth_hash(self.request, instance)` to prevent users from getting logged out.

Reply:
Nevermind, I see it above in the FacilityUserViewSet. But this is for the sign-up flow.